handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -61,29 +61,25 @@
         // If in the classpath, try to load the native library and initialize netty-tcnative.
         if (cause == null) {
             try {
-                String os = normalizeOs(SystemPropertyUtil.get("os.name", ""));
-                String arch = normalizeArch(SystemPropertyUtil.get("os.arch", ""));
-
-                Set<String> libNames = new LinkedHashSet<String>(3);
-                // First, try loading the platform-specific library. Platform-specific
-                // libraries will be available if using a tcnative uber jar.
-                libNames.add("netty-tcnative-" + os + '-' + arch);
-                if (LINUX.equalsIgnoreCase(os)) {
-                    // Fedora SSL lib so naming (libssl.so.10 vs libssl.so.1.0.0)..
-                    libNames.add("netty-tcnative-" + os + '-' + arch + "-fedora");
-                }
-                // finally the default library.
-                libNames.add("netty-tcnative");
-
-                NativeLibraryLoader.loadFirstAvailable(SSL.class.getClassLoader(),
-                        libNames.toArray(new String[libNames.size()]));
-
-                Library.initialize("provided");
-                SSL.initialize(null);
+                // The JNI library was not already loaded. Load it now.
+                loadTcNative();
             } catch (Throwable t) {
                 cause = t;
                 logger.debug(
-                        "Failed to load netty-tcnative; " +
+                    "Failed to load netty-tcnative; " +
```

Comment:
@carl-mastrangelo not sure I follow.  This can happen if the library was previously loaded, in which case this error is expected (and would be noise if the log level was higher). Or am I missing something?